src/main/java/frc/robot/Constants.java

```diff
@@ -4,16 +4,39 @@
 
 package frc.robot;
 
+import edu.wpi.first.units.Distance;
+import static edu.wpi.first.units.Units.*;
+import edu.wpi.first.units.Measure;
+import edu.wpi.first.units.Velocity;
+import edu.wpi.first.units.Angle;
 /**
- * The Constants class provides a convenient place for teams to hold robot-wide numerical or boolean
- * constants. This class should not be used for any other purpose. All constants should be declared
+ * The Constants class provides a convenient place for teams to hold robot-wide
+ * numerical or boolean
+ * constants. This class should not be used for any other purpose. All constants
+ * should be declared
  * globally (i.e. public static). Do not put anything functional in this class.
  *
- * <p>It is advised to statically import this class (or one of its inner classes) wherever the
+ * <p>
+ * It is advised to statically import this class (or one of its inner classes)
+ * wherever the
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
   public static class OperatorConstants {
     public static final int kDriverControllerPort = 0;
   }
-}
+
+  // TODO: shooter constants
+  public static class ShooterConstants {
+    public static final int feederMotorCANID = 9;
+    public static final int kickerMotorCANID = 10;
+    public static final int shooterRightCANID = 11;
+    public static final int shooterLeftCANID = 12;
```

Comment:
Can these CANID's please be changed to the following:
feederMotorCANID = 10;
kickerMotorCANID = 11;
shooterRightCANID = 12;
shooterLeftCANID = 13;

Also, is there a reason why the pivot motors are not declared here?